.github/workflows/build_packages.yaml

```diff
@@ -341,14 +341,6 @@ jobs:
         arch:
           - [amd64, x86_64]
           - [arm64v8, aarch64]
-          - [arm32v7, arm]
```

Comment:
Worth to keep it? Since quicer can support it and the majority of raspberrypi are running this arch I think.

Reply:
If the quic can support armv7, let's keep it